CHANGES.md

```diff
@@ -4,6 +4,8 @@
 
   + Add an option `--format-invalid-files` to print unparsable parts of the input as verbatim text. This feature is still experimental. (#1026) (Guillaume Petiot)
 
+  + Support multi-indices extended indexing operators (#1277) (Guillaume Petiot)
```

Comment:
Could you say that it's a new feature from 4.10 ?

Reply:
done